packages/db/test/utils.js

```diff
@@ -119,3 +122,13 @@ export async function getUpload (dbClient, cid, userId) {
 export async function getPinSyncRequests (dbClient, size = 10) {
   return dbClient.getPinSyncRequests({ size })
 }
+
+/**
+ * @param {number} code
+ * @returns {Promise<string>}
+ */
+export async function randomCid (code = pb.code) {
+  const bytes = crypto.getRandomValues(new Uint8Array(10))
+  const hash = await sha256.digest(bytes)
+  return CID.create(1, code, hash).toString()
+}
```

Comment:
Minor thing, but we've already got a `randomCid` function in `db/test/pinning.spec.js`. So maybe that one can be removed and replaced with an import of this one?

Also, given that it's just for testing, and therefore doesn't need to be _actually_ random in a secure way, it could just use `Math.random()` which would be faster than the cryptographically-secure `crypto.getRandomValues`. Every little helps! 🤷  See [randomCid in api package](https://github.com/web3-storage/web3.storage/blob/main/packages/api/test/scripts/helpers.js#L62-L65).

Reply:
Amended, and refactored `pinning-spec.js` to use the same.

Reply:
Sweet